src/ui/common/package.json

```diff
@@ -128,5 +127,8 @@
     "vite": "^4.0.4",
     "vite-plugin-externalize-deps": "^0.4.0",
     "webpack": "^5.74.0"
+  },
+  "dependencies": {
```

Comment:
This should go with the rest of our dependencies in the peerDependencies object.